client/src/Components/Footer/Footer.module.css

```diff
@@ -1,33 +1,42 @@
 .footer {
-  background-color: #E6E6E6;
+  width: 100vw;
+  height: 219px;
+  background: rgb(10,14,36);
+  background: linear-gradient(90deg, rgba(10,14,36,1) 0%, rgba(0,64,87,1) 90%);
```

Comment:
Is there a need for the first background property? It gets overridden by the second one anyways?